app/controllers/events/event_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Events::EventController < ActionController::Base
+  protect_from_forgery with: :null_session
+
+  def create
+    event = Event.new(event_params)
+    if event.save
+      render json: { message: 'Event created' }, status: :created
+    else
+      render json: { message: event.errors.full_messages }, status: :bad_request
```

Comment:
i don't know enough about the event model but should we guard against `nil` in this accessor? not sure if we are guaranteed to get `errors` back on the event. Correct me if I'm wrong though.

Reply:
I am confused what you are asking? check for nil on event.errors? to my knowledge if event.save returns false, event.errors should be initialized

Reply:
on second thought, can we avoid this conditional and throw if the record fails to save? we have similar looking code in the rescue clause below. I was reading up on Model.save and it looks like `save` will return a boolean and `save!` will throw an exception if it fails. Maybe we could switch to using the bang method and rely on the rescue to catch any errors?

Reply:
Why not make use of the more readable errors that .save would allow? Exception messages can be messy imo.